app/Http/Controllers/EmployeeController.php

```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers;
 use App\Models\Employee;
+use Dotenv\Parser\Value;
```

Comment:
Please explain why have you used the `Parser` @axnish27

Reply:
Sorry it was added by mistake...ill remove

Reply:
@Raza9798  removed unnecessary files please check